.github/ISSUE_TEMPLATE.md

```diff
@@ -13,20 +13,25 @@ is the best place for getting support.
 
 [Description of the issue]
```

Comment:
✏️ I think we should be more clear with a comment in this section explaining that in the case of a feature/enhancement that we're looking for answers to _Why_ the change is being requested/suggested. A justification and/or use case would be helpful.

It's probably also a good place to explicitly link to the GH wiki docs on writing an issue.

Reply:
https://www.atlassian.com/blog/archives/effective_jira_issues
This last part is kind of interesting:
>h4. When Filing a Feature Request
Detail, detail, detail, context, context, context. 
Tell us where the feature request came from, as well as what it is. Tell us _why_ you want the feature. If the request comes from an email or IM exchange between co-workers, paste that into the issue too!
Knowing why a somebody wants something done is almost as important, in terms of scheduling and planning features, as knowing what they want. Sometimes it’s more important...